example/src/MyComponent.jsx

```diff
@@ -1,6 +1,8 @@
 import React, { useState } from 'react';
-import { Button, Form, Icon, Bubble, Stack, Container } from '@openedx/paragon'; // eslint-disable-line
-import { FavoriteBorder } from '@openedx/paragon/icons'; // eslint-disable-line
+// eslint-disable-next-line import/no-extraneous-dependencies, object-curly-newline
+import { Button, Form, Icon, Bubble, Stack, Container } from '@openedx/paragon';
```

Comment:
We can most likely remove the `object-curly-newline` in favor of the following:

```suggestion
import {
  Button, Form, Icon, Bubble, Stack, Container
} from '@openedx/paragon';
```

But it is good to be explicit about the `import/no-extraneous-dependencies` rule vs. the `eslint-disable-line`.

Reply:
Good catch! Removed unnecessary `object-curly-newline` rule
https://github.com/openedx/paragon/pull/3488/files#diff-5dcc0fb0dadec970ffd28d9d4cd5cdc6dd928f2edbf682be2df77bf1256320eeR2